dev/Mock/MockGateway.php

```diff
@@ -308,6 +313,9 @@ private function createNewAssertion($nameId, $authnContextClassRef, $destination
             'Format' => Constants::NAMEID_UNSPECIFIED,
             'Value' => $nameId,
         ]);
+        $newAssertion->setAttributes([
+            'urn:mace:dir:attribute-def:emailAddress' => [$emailAddress]
```

Comment:
As Thijs pointed out, this urn:mace attribute is not in the dictionary. According to the page linked it should be; `urn:mace:dir:attribute-def:mail` This was also incorrectly fixed in `AzureMfaService.php`. There it now reads `email`

Reply:
Changed it to be up to the user, now a custom set of attributes could be set win the MockIdP  so this is also better testable.